### DISCUSSION TITLE
Is my pointcloud GNN Regression task too easy?

### DISCUSSION THREAD

**Discussion by davodogster** [pyg-team/pytorch_geometric#5619] [Q&A]
Hi guys,
I am not using any dropout or topKpooling or dropping edges and it seems to be learning quite well. Does it mean the val/testing data is too similar/easy compared with my training data? It's a point cloud KNN Graph Regression task (Response values range from 0 to 100 (step size = 10)
![image](https://user-images.githubusercontent.com/46079516/194220359-e39cd70d-a394-47a7-8aec-ee3c2951f24b.png)

![image](https://user-images.githubusercontent.com/46079516/194220423-68afc322-b5a1-458f-862d-afd3d19aa1ae.png)

Thanks, Sam

**Comment by rusty1s** [comment]
This is hard to tell. It may just mean that examples in your dataset are indeed quite similar across the different splits.
As long as you are happy about the results there is nothing to worry about :)
